docs/content/tutorials/firewall.md

```diff
@@ -0,0 +1,189 @@
+                               "newPage",
+                               "robot",
+                               "anonymous",
+                               "namespace",
+                               "continent",
+                               "country",
+                               "region",
+                               "city"
+                           ],
+                           "dimensionExclusions": [],
+                           "spatialDimensions": []
+                       }
+                   }
+               },
+               "metricsSpec": [
+                   {
+                       "type": "count",
+                       "name": "count"
+                   },
+                   {
+                       "type": "doubleSum",
+                       "name": "added",
+                       "fieldName": "added"
+                   },
+                   {
+                       "type": "doubleSum",
+                       "name": "deleted",
+                       "fieldName": "deleted"
+                   },
+                   {
+                       "type": "doubleSum",
+                       "name": "delta",
+                       "fieldName": "delta"
+                   }
+               ],
+               "granularitySpec": {
+                   "type": "uniform",
+                   "segmentGranularity": "DAY",
+                   "queryGranularity": "NONE"
+               }
+           },
+           "ioConfig": {
+               "type": "realtime",
+               "firehose": {
+                   "type": "kafka-0.8",
+                   "consumerProps": {
+                       "zookeeper.connect": "localhost:2181",
+                       "zookeeper.connection.timeout.ms": "15000",
+                       "zookeeper.session.timeout.ms": "15000",
+                       "zookeeper.sync.time.ms": "5000",
+                       "group.id": "druid-example",
+                       "fetch.message.max.bytes": "1048586",
+                       "auto.offset.reset": "largest",
+                       "auto.commit.enable": "false"
+                   },
+                   "feed": "wikipedia"
+               },
+               "plumber": {
+                   "type": "realtime"
+               }
+           },
+           "tuningConfig": {
+               "type": "realtime",
+               "maxRowsInMemory": 500000,
+               "intermediatePersistPeriod": "PT10m",
+               "windowPeriod": "PT10m",
+               "basePersistDirectory": "/tmp/realtime/basePersist",
+               "rejectionPolicy": {
+                   "type": "messageTime"
+               }
+           }
+       }
+    ]
+    ```
+
+3. Refer to the [Running Example Scripts](#running-example-scripts) section to start the example Druid Realtime node by issuing the following from within your Druid directory:
+
+    ```bash
+    ./run_example_server.sh
+    ```
+
+#### Kafka Setup and Configuration
```

Comment:
This should link to some other page that describes how to get Kafka running with Druid. This information is valuable to a lot of people, not just people that have a firewall blocking the internet.